package-lock.json

```diff
@@ -48119,11 +48119,18 @@
       }
     },
     "sha3": {
-      "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/sha3/-/sha3-1.2.0.tgz",
-      "integrity": "sha1-aYnxtwpJhwWHajc+LGKs6WqpOZo=",
+      "version": "1.2.3",
```

Comment:
It seems that `sha3` was the only incompatible package. The `1.2.3` maintenance release supports up to Node v12 though.